src/pages/SignUpPage.js

```diff
@@ -6,6 +6,9 @@ import { useSession, useDispatch } from 'hooks';
 import { signUp } from 'state/actions/userActions';
 import SignUpForm from 'components/user/SignUpForm';
 import routes from 'constants/routesPaths';
+import AuthWrapper from '../components/common/AuthWrapper';
```

Comment:
Try not to use relative paths in these cases, `components/common/AuthWrapper` should work